### ISSUE TITLE
Use Snap Packaging and Submit to Ubuntu Software Store

### ISSUE THREAD

**Issue by ryanleesipes** [BoostIO/Boostnote#379]
If we use snap packaging (Ubuntu's new universal packaging standard), we can submit this app to Ubuntu's developer portal and have it show up in the software store.

**Comment by smparekh** [comment]
Any update on this, looks like the snap yaml files are pulled in and its ready for packaging but I don't see it available if I do a `snap find boostnote`

**Comment by peey** [comment]
Yeah, searching for "boostnote snap" leads me to [this](https://github.com/BoostIO/Boostnote/tree/master/snap) which made me think it's already available on snap, but it isn't

**Comment by Flexo013** [comment]
#1514 is a PR that creates a snap package. Feel free to continue that PR and submit a new one.

**Comment by Flexo013** [comment]
Related: #2115 #2164
